cumulus/node/README.md

```diff
@@ -7,105 +7,68 @@
+2022-04-24 17:00:28.094  INFO main sub-libp2p: [PrimaryChain] 🏷  Local node identity is: 12D3KooWEna4n2m3B6EKXQE1jZhQ5sfYcr9TpVmr8Yk9S8zCpnm4
+2022-04-24 17:00:28.096  INFO main subspace: [PrimaryChain] 🧑‍🌾 Starting Subspace Authorship worker
+2022-04-24 17:00:28.099  INFO main sc_sysinfo: [PrimaryChain] 💻 Operating system: macos
+2022-04-24 17:00:28.099  INFO main sc_sysinfo: [PrimaryChain] 💻 CPU architecture: aarch64
+2022-04-24 17:00:28.099  INFO main sc_service::builder: [PrimaryChain] 📦 Highest known block at #0
+2022-04-24 17:00:28.099  INFO tokio-runtime-worker substrate_prometheus_endpoint: [PrimaryChain] 〽️ Prometheus exporter started at 127.0.0.1:9615
+2022-04-24 17:00:28.100  INFO                 main parity_ws: [PrimaryChain] Listening for new connections on 127.0.0.1:9944.
+2022-04-24 17:00:28.100  WARN                 main sc_cli::commands::run_cmd: [SecondaryChain] Running in --dev mode, RPC CORS has been disabled.
+2022-04-24 17:00:28.235  INFO                 main sc_service::client::client: [SecondaryChain] 🔨 Initializing Genesis block/state (state: 0x8e63…66a2, header-hash: 0x35d4…5e4f)
+2022-04-24 17:00:28.235  WARN                 main sc_service::config: [SecondaryChain] Using default protocol ID "sup" because none is configured in the chain specs 
+2022-04-24 17:00:28.236  INFO                 main sub-libp2p: [SecondaryChain] 🏷  Local node identity is: 12D3KooWLDx1XEAyDWoxtJZhEj9WBspb8C9BQbyS7x4n6qoAFsAZ
+2022-04-24 17:00:28.326  INFO                 main sc_sysinfo: [SecondaryChain] 💻 Operating system: macos
+2022-04-24 17:00:28.326  INFO                 main sc_sysinfo: [SecondaryChain] 💻 CPU architecture: aarch64
+2022-04-24 17:00:28.326  INFO                 main sc_service::builder: [SecondaryChain] 📦 Highest known block at #0
+2022-04-24 17:00:28.326  INFO tokio-runtime-worker substrate_prometheus_endpoint: [SecondaryChain] 〽️ Prometheus exporter started at 127.0.0.1:9616
+2022-04-24 17:00:28.326  INFO                 main parity_ws: [SecondaryChain] Listening for new connections on 127.0.0.1:8846.
 ```
 
-Note the `Local node identity`(`12D3KooWRreNzoMVgM6HtPVP27enDaAuPuPbYgGCrSr2RWD8UBGf`) from the log output. the embedded primary node will use it to craft a bootnode for connecting to the primary node. You can also directly retrieve the primary peer id using the RPC `system_localPeerId`.
+Note the `Primary node identity`(`12D3KooWEna4n2m3B6EKXQE1jZhQ5sfYcr9TpVmr8Yk9S8zCpnm4`) from the log output. We'll start another primary node running an executor full node in next step and will use it to as a bootnode to connect to this primary node. You can also directly retrieve the primary peer id using the RPC `system_localPeerId`.
 
 Start a farmer:
 
 ```bash
-$ ./target/release/subspace-farmer wipe && ./target/release/subspace-farmer farm
+$ ./target/release/subspace-farmer wipe && ./target/release/subspace-farmer farm --reward-address REWARD_ADDRESS --plot-size 10G
```

Comment:
Can be convenient if we support the well-known addresses like `--reward-address Alice` for the local development, what do you think? @nazar-pc